app/api/orders.py

```diff
@@ -68,12 +68,13 @@ def is_payment_valid(order, mode):
         return (order.paid_via == 'paypal') and order.transaction_id
 
 
-def check_billing_info(data):
-    if data.get('amount') and data.get('amount') > 0 and not data.get('is_billing_enabled'):
+def check_billing_info(data, order):
+	if data.get('amount') and data.get('amount') > 0 and order.event.is_billing_info_mandatory:
```

Comment:
@kushthedude how is this a valid check? You are just checking amount and `is_billing_info_mandatory` here.

Reply:
Do anything more needs to be get checked for paid order?

Reply:
And the travis is failing with a proper message - `TabError: inconsistent use of tabs and spaces in indentation`. Please take a look and fix it

Reply:
> Do anything more needs to be get checked for paid order?

So, if an order is paid and it has `is_billing_info_mandatory` you're raising an error. Obv you should check if billing data is missing or not before raising an error

Reply:
Then I think I should merge this check in the second one which contains the missing information error .

Reply:
Yes, exactly

Reply:
Done